test/powershell/Host/Startup.Tests.ps1

```diff
@@ -93,7 +93,7 @@ Describe "Validate start of console host" -Tag CI {
             Remove-Item $profileDataFile -Force
         }
 
-        $loadedAssemblies = & "$PSHOME/pwsh" -noprofile -command '([System.AppDomain]::CurrentDomain.GetAssemblies()).manifestmodule | Where-Object { $_.Name -notlike ""<*>"" } | ForEach-Object { $_.Name }'
```

Comment:
For scripts that currently use double-double-quotes, will they now start failing with this PR?

Reply:
This is a case of over-escaping, and it occurs as a native-command call. So yes, it will fail, since it is literally now passing the wrong command 

```pwsh
([System.AppDomain]::CurrentDomain.GetAssemblies()).manifestmodule | Where-Object { $_.Name -notlike ""<*>"" } | ForEach-Object { $_.Name }
```

Reply:
I understand that, but the problem is that it's very likely existing production scripts are relying on that behavior.  So "fixing" it will break them.